granary/tests/test_rss.py

```diff
@@ -203,16 +203,18 @@ def test_author(self):
         'author': {
           'objectType':'person',
           'displayName':'Mrs. Baz',
+          'email': 'baz@example.com',
         },
       }], feed_url='http://this')
-    self.assert_multiline_in('<author>- (Mrs. Baz)</author>', got)
+    self.assert_multiline_in('<author>baz@example.com (Mrs. Baz)</author>', got)
 
   def test_author_string_id(self):
     got = rss.from_as1([{
         'content': 'foo bar',
-        'author': 'tag:bob',
+        'author': 'tag:bob',  # should be ignored for RSS
+        'email': 'bob@example.com',
```

Comment:
minor nit, remove this line? it's not used in this test, and it's a bit confusing as is, since it would go into the `author` object, not at the top level.

Reply:
oh right, i'd forgotten to clean that up. thanks!